docs/datadoc/keywords.md

```diff
@@ -56,6 +56,8 @@ Resource published or curated by an agent.
 | [type]                  | [skos:Concept]                                 | optional    | A type of the resource.                                                                                                                      | A recommended controlled vocabulary data-type is foreseen.                                                                                                                                            |
 | [version]               | [rdfs:Literal]                                 | optional    | Version indicator (name or identifier) of a resource.                                                                                        |                                                                                                                                                                                                       |
 | [versionNotes]          | [rdfs:Literal]                                 | optional    | A description of the differences between this version and a previous version of the resource.                                                | This property can be repeated for parallel language versions of the version notes.                                                                                                                    |
+| [hasCurationDate]       | [rdfs:Literal]<br>(xsd:date)                   |             | The date the resource was curated.                                                                                                           | Should be used with `hasCurator`.                                                                                                                                                                     |
```

Comment:
I am not convinced about hasCurationDate. This is something that refers back to a day in the past.

How about wasCuratedOn and wasCuratedBy, to identify who did what when. However these have to be linked somehow. Can this be done simply with triples?.

hasCurator I would think is the person(s) who are still in charge of the data?

Reply:
Replace with `curatedBy` and `curationDate`